backend/lib/services/members/MemberManager.js

```diff
@@ -169,6 +184,21 @@ class MemberManager {
     this.subjectList.delete(item.id)
   }
 
+  async deleteServiceAccountSecret (item) {
+    const { namespace } = Member.parseUsername(item.id)
+    if (namespace !== this.namespace) {
+      return
+    }
+
+    const secretName = _
+      .chain(item)
+      .get('extensions.secrets')
+      .head()
```

Comment:
What if we have more than one secret?

Reply:
Hmm, well I asked myself the same question. The way we generate kubeconfigs is that we take the first secret so I guessed, deleting the first one will do the job. I don't know how we should handle service accounts with multiple secrets. We could of course delete all, but I don't know if this what we want. WDYT?

Reply:
we can throw an error if there is more than one secret (422)